<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>find_dialog</class>
 <widget class="QDialog" name="find_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>356</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Find Orientation</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="ScientificDoubleSpinBox" name="angular_tolerance">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Default: 0.07°&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="suffix">
      <string>°</string>
     </property>
     <property name="decimals">
      <number>8</number>
     </property>
     <property name="maximum">
      <double>100000.000000000000000</double>
     </property>
     <property name="value">
      <double>0.070000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="conserve_memory">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If checked, a different version of the internal function will be utilized that uses less RAM, but also runs a little slower.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Conserve Memory</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="resolution_limit_label">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Default: 0.4 Å&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Resolution Limit:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="ScientificDoubleSpinBox" name="resolution_limit">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Default: 0.4 Å&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="suffix">
      <string> Å</string>
     </property>
     <property name="decimals">
      <number>8</number>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
     <property name="value">
      <double>0.400000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="angular_tolerance_label">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Default: 0.07°&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Angular Tolerance:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="button_box">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="note_label">
     <property name="text">
      <string>Note: you may still manually add or remove points while this dialog is open.</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="reflections_threshold_label">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The minimum number of indexed reflections to assume the indexing routine was successful.&lt;br/&gt;&lt;/p&gt;&lt;p&gt;Default: 3&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Reflections Threshold:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="cell_parameters_group">
     <property name="title">
      <string>Cell Parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="a_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>a:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="ScientificDoubleSpinBox" name="a">
        <property name="suffix">
         <string> Å</string>
        </property>
        <property name="decimals">
         <number>8</number>
        </property>
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
        <property name="value">
         <double>5.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="b_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>b:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="ScientificDoubleSpinBox" name="b">
        <property name="suffix">
         <string> Å</string>
        </property>
        <property name="decimals">
         <number>8</number>
        </property>
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
        <property name="value">
         <double>5.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="c_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>c:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="ScientificDoubleSpinBox" name="c">
        <property name="suffix">
         <string> Å</string>
        </property>
        <property name="decimals">
         <number>8</number>
        </property>
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
        <property name="value">
         <double>5.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="alpha_label">
        <property name="text">
         <string>α:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="ScientificDoubleSpinBox" name="alpha">
        <property name="suffix">
         <string>°</string>
        </property>
        <property name="decimals">
         <number>8</number>
        </property>
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
        <property name="value">
         <double>90.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="beta_label">
        <property name="text">
         <string>β:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="ScientificDoubleSpinBox" name="beta">
        <property name="suffix">
         <string>°</string>
        </property>
        <property name="decimals">
         <number>8</number>
        </property>
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
        <property name="value">
         <double>90.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="gamma_label">
        <property name="text">
         <string>λ:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="ScientificDoubleSpinBox" name="gamma">
        <property name="suffix">
         <string>°</string>
        </property>
        <property name="decimals">
         <number>8</number>
        </property>
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
        <property name="value">
         <double>90.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="reflections_threshold">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The minimum number of indexed reflections to assume the indexing routine was successful.&lt;br/&gt;&lt;/p&gt;&lt;p&gt;Default: 3&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
     <property name="value">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScientificDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>scientificspinbox.py</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>a</tabstop>
  <tabstop>b</tabstop>
  <tabstop>c</tabstop>
  <tabstop>alpha</tabstop>
  <tabstop>beta</tabstop>
  <tabstop>gamma</tabstop>
  <tabstop>angular_tolerance</tabstop>
  <tabstop>resolution_limit</tabstop>
  <tabstop>reflections_threshold</tabstop>
  <tabstop>conserve_memory</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>find_dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
